frontend/src/metabase/lib/colors/groups.ts

```diff
@@ -73,7 +73,7 @@ export const getPreferredColor = (key: string, palette?: ColorPalette) => {
     case "unstable":
       return color("warning", palette);
     case "count":
-      return color("brand", palette);
+      return color("accent0", palette);
```

Comment:
It might not be apparent from the [product doc](https://www.notion.so/metabase/Appearance-Settings-page-polish-f9b02b47a0ae427ba8be810dfa924e0d?pvs=4#47fc44afb0f04a1d8e16eaf653a9859e), but we've clarified this in the kick-off that we don't want application colors to affect the chart colors.

See also the [Slack discussion](https://metaboat.slack.com/archives/C063Q3F1HPF/p1707901949108579)